server.py

```diff
@@ -16,6 +16,8 @@
 # Utilizes CORS with the list of origin strings and regex expressions to validate resource requests.
 CORS(app, origins=origin_list)
 
+restrictedTables = ['archive', 'txn_history']
+nonRestrictedTables = ['intake', 'metadata', 'violations', 'records']
```

Comment:
The /adminlist endpoint shouldn't be prevented from fetching non-restricted tables. You can either concatenate both of these lists for /adminlist's check (creating a master list) or just skip the check for /adminlist and let that route query whatever table it wants to

Reply:
Fixed - deleted table checking